### PR TITLE
Added Routing and basic UI 

### PR DESCRIPTION
**Pull request by shobit1337**
Created Routes for different pages:
- Homepage
- PlaylistPage
- LikesPage
- Watch Later Page
- History Page

Basic UI and styling added.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+:root {
+  --primary-color: #320349;
+  --secondary-color: #ecbc0a;
+  /* --accient-color: */
```
Comment (ScoobyDubeyDo) [inline src/App.css:4]: **suggestion**: delete unused code

File: src/components/VideoCard/VideoCard.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.video-card {
+  display: flex;
+  position: relative;
+  min-width: 250px;
+  max-width: 250px;
```
Comment (ScoobyDubeyDo) [inline src/components/VideoCard/VideoCard.css:5]: **suggestion**:-  use relative units in place of pixels

**Review by ScoobyDubeyDo — APPROVED** [review]
The code looks clean and good, but in a few places there is unused code (comments), please remove them in your next commit.

**Review by Ash-D23 — APPROVED** [review]
looks good, liked the way you have used Nested Routing

File: src/components/VideoCard/VideoCard.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <span className='action-btn'>
+          <i class='far fa-clock'></i>
+        </span>
+      </div>
+      <div className='video-thumbnail'>
+        <img src='https://picsum.photos/250/150' alt='' />
+      </div>
+      <div className='footer'>
+        <div className='title'>The Batman</div>
+        {/* <div className='card-creator'>Shobit Deshwal</div> */}
```
Comment (Vikas2426) [inline src/components/VideoCard/VideoCard.jsx:20]: should not commit commented code

**Review by Vikas2426 — APPROVED** [review]

**Pull request merged**